test/utils/Address.test.js

```diff
@@ -138,6 +140,7 @@ describe('Address', function () {
       });
 
       it('reverts when the called function runs out of gas', async function () {
+        if (coverage) { return this.skip(); }
```

Comment:
What's the reason we have to do this? Can you add a comment explaining that here?

Reply:
This test attempts to make the execution run out of gas, but the coverage EVM sets a ridiculously high gas limit. In practice, this ends up timeouting and causing all other tests to fail due to the hang in Ganache.

Reply:
It is due to coverage setting block gas limit to `0xffffffffff` which cause `mockFunctionOutOfGas` function to crash Ganache and the subsequent tests before running out of gas.
```
  function mockFunctionOutOfGas() public payable {
        for (uint256 i = 0; ; ++i) {
            _array.push(i);
        }
    }
```